core/src/test/java/org/elasticsearch/search/suggest/AbstractSuggestionBuilderTestCase.java

```diff
@@ -211,6 +213,22 @@ public void testFromXContent() throws IOException {
         }
     }
 
+    protected Tuple<MapperService, SB> mockMapperServiceAndSuggestionBuilder(
```

Comment:
Nevermind, I just got to the part where this is overwritten in CompletionSuggesterBuilderTests. Quiet tricky though, I wonder if this could be made somewhat easier to understand, haven't completely wrapped my head around it yet.